src/mono/wasm/threads.md

```diff
@@ -0,0 +1,80 @@
+# Threaded runtime #
+
+## Building ##
+
+Build with `/p:WasmEnableThreads=true` to enable support for multi-threading.
```

Comment:
Will this also enable EventPipe diagnostics support?

Reply:
I think so.  Once we have full user-visible multithreading there's no particular reason to exclude eventpipe.